src/fieldToConceptIdMapping.js

```diff
@@ -1,4 +1,4 @@
-export default {
+export const conceptIds = {
```

Comment:
Will this changed export from `default` to `conceptIds` break imports in other files?
I'd suggest putting `export default conceptIds;` at end of this file, to avoid breaks.

Reply:
I see imports are updated based on changed export. So nothing should break.